src/push_service/keys.rs

```diff
@@ -45,6 +45,52 @@ impl PushService {
         .map_err(Into::into)
     }
 
+    /// Checks for consistency of the repeated-use keys
+    ///
+    /// Supply the digest as follows:
+    /// `SHA256(identityKeyBytes || signedEcPreKeyId || signedEcPreKeyIdBytes || lastResortKeyId ||
+    /// lastResortKeyBytes)`
+    ///
+    /// The IDs are represented as 8-byte big endian ints.
+    ///
+    /// Retuns `Ok(())` if the view is consistent.
```

Comment:
This is not quite correct; `Ok(true)` here I think, and what does `Ok(false)` mean?